### DISCUSSION TITLE
When is the next release?

### DISCUSSION THREAD

**Discussion by anhlinh123** [rui314/mold#1109] [General]
Thank you for creating this great linker. We've been trying our best to integrate this into our stack, and we're all eager to be able to use it. The current release misses some of the important fixes that are essential to our integration.
May I ask when the next release will be, so we can plan our projects accordingly?
If possible, could you make a new release soon?

**Comment by rui314** [comment]
I can make a new release, I just thought that we didn't have enough number of important changes to make another release at this point. What changes did you want to see in the next release?

**Reply by anhlinh123** [reply]
We need this https://github.com/rui314/mold/commit/f02db0fa0f6fef9e25ccc168f68c81830dd4ed98

**Reply by rui314** [reply]
I'll make a new release for you. Please wait for a few days.

**Reply by anhlinh123** [reply]
Thank you very much!

**Reply by rui314** [reply]
I just released v2.2.0.
